Simplenote/src/main/java/com/automattic/simplenote/widgets/ImageButtonBehavior.java

```diff
@@ -0,0 +1,51 @@
+package com.automattic.simplenote.widgets;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.View;
+import android.widget.ImageButton;
+
+import androidx.annotation.NonNull;
+import androidx.coordinatorlayout.widget.CoordinatorLayout;
+
+import com.google.android.material.snackbar.Snackbar;
+
+import java.util.List;
+
+public class ImageButtonBehavior extends CoordinatorLayout.Behavior<ImageButton> {
```

Comment:
Can you elaborate on the use of this class?
I understand from its code that it tries to move the "add" button if a snackbar is displayed, but from what I see, it doesn't do anything in the current state as the ImageButton it's applied to is not a child of a CoordinatorLayout.
And a question, do we actually need it? as I didn't find any way to trigger a SnackBar when viewing TagsActivity.

Reply:
You're right.  This isn't utilized at the moment.  It was included to future-proof the case when a snackbar was used on the ***Edit Tags*** screen like for an ***Undo*** action when deleting a tag.

Reply:
IMO it's better to not add it to the code until the undo feature is added, as the changes are more than just the behavior: CoordinatorLayout should be used too.
And the least is to add a comment explaining that it's not used now.

Reply:
I removed the `ImageButtonBehavior` class in ef90cdd6.

Reply:
Sorry for nitpicking, but the ImageButton has still the app:layout_behavior property :).

I'll approve the PR, you can merge it when it's removed

Reply:
No need to apologize.  That's why code reviews exist.  We shouldn't leave that attribute in the code if the class doesn't exist.  I removed the attribute in 5b4a969.